cpp-client/deephaven/client/src/client.cc

```diff
@@ -558,6 +591,425 @@ std::string TableHandle::toString(bool wantHeaders) const {
   return std::move(oss.str());
 }
 
+UpdateByOperation::UpdateByOperation() = default;
```

Comment:
Same comment than in client.h; potentially asking to consider separating the updateBy stuff to its own `.cc` file.

Reply:
done